src/utils/jquery.js

```diff
@@ -11,15 +11,17 @@ if (window.jQuery) jQuery = window.jQuery
 else if (_.get(window, '$.prototype.jquery')) jQuery = window.$
 
 if (jQuery) {
-  throw new Error([
+  /* eslint-disable no-console */
+  console.error([
     'Do not load jQuery, it is bundled with Stardust.',
     'Stardust will soon be jQuery free: https://github.com/TechnologyAdvice/stardust/issues/247.',
   ].join(' '))
+  /* eslint-enable no-console */
 }
 
 debug('Loading jQuery')
 // load jQuery, then load SUI jQuery
-window.jQuery = window.$ = require('jquery')
+window.jQuery = window.$ = jQuery || require('jquery')
```

Comment:
Prevents overriding `window.jQuery` if it exists, which was the source of jQuery extension issues in the past.